src/coprocessor/endpoint.rs

```diff
@@ -173,10 +173,15 @@ impl RequestTask {
         COPR_REQ_HANDLE_TIME.with_label_values(&["select", type_str])
             .observe(handle_time - wait_time);
 
-        let scanned_keys = self.statistics.total_op_count() as f64;
-        COPR_SCAN_KEYS.with_label_values(&["select", type_str]).observe(scanned_keys);
-        let inefficiency = self.statistics.inefficiency();
-        COPR_SCAN_INEFFICIENCY.with_label_values(&["select", type_str]).observe(inefficiency);
+        for (cf, total_op_count) in self.statistics.total_op_count() {
+            COPR_SCAN_KEYS.with_label_values(&["select", type_str, cf])
+                .observe(total_op_count as f64);
+        }
+
+        for (cf, inefficiency) in self.statistics.inefficiency() {
```

Comment:
I prefer not using inefficiency now, we can report total and matched keys separately.